server/command.go

```diff
@@ -276,6 +308,9 @@ func getAutocompleteData() *model.AutocompleteData {
 	send.AddTextArgument("Todo message", "[message]", "")
 	todo.AddCommand(send)
 
+	settings := model.NewAutocompleteData("settings", "summary [on] [off]", "Sets the user preference on daily reminder")
```

Comment:
Maybe out of scope of the PR: It would be great if there are autocomplete options for `on` and `off`

Reply:
Sure, I can cover in this PR.